pkg/transform/fixtures/single_container_object.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2017 The Kedge Authors All rights reserved.
```

Comment:
Can you rabse with current master? 
This diff looks weird. `single_container_object.go` file shouldn't even exist after #180 being merged